### DISCUSSION TITLE
Lost CSS When Refresh Page

### DISCUSSION THREAD

**Discussion by ducanhkatori00** [serenity-is/Serenity#5717] [Help]
Hello guys,
I add css class but when I refresh page it lost.
Please help me. Thanks in advanced.

This is image before refresh page
![image](https://user-images.githubusercontent.com/77576309/118214467-f5193d00-b499-11eb-8303-f1b6e618fc10.png)

This is image after refresh page
![image](https://user-images.githubusercontent.com/77576309/118214531-15e19280-b49a-11eb-8bd1-9547b023424f.png)

This is the code
![image](https://user-images.githubusercontent.com/77576309/118214623-4295aa00-b49a-11eb-951b-b2faf5a51db1.png)

**Comment by minhhungit** [comment]
of course
you should handle the feature by adding condition in **getItemCssClass**

check this sample check this sample https://serenity.is/demo/BasicSamples/ConditionalFormatting

Code here
https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/BasicSamples/Grids/ConditionalFormatting/ConditionalFormattingGrid.ts#L40

**Reply by ducanhkatori00** [reply]
Thanks. But I only add class for Status field = Approve or Decline. Help me!

**Reply by minhhungit** [reply]
Then your code should be something like this ( put this into your grid.ts ):

```javascript
protected getItemCssClass(item: Northwind.ProductRow, index: number): string {
      let klass: string = "";

      if (item.Status== "Approve" || item.Status == "Decline" )
          klass += " approve-or-decline";
      
      return Q.trimToNull(klass);
  }
```

And declare css class `approve-or-decline` into site.less or the cshtml page.

The above code will  add class `approve-or-decline` when you load grid data. So after calling "HrAdsendService.ApproveMessage" to update status and refresh page you won't lost the css

**Comment by VictorTomaili** [comment]
@ducanhkatori00 Please don't use letters as all capital in title. And check [this](https://en.wikipedia.org/wiki/All_caps) about why shouldn't you use all caps.

**Reply by ducanhkatori00** [reply]
oke bro, i'm sorry
